test/util/all_close_f.cpp

```diff
@@ -82,8 +94,20 @@ uint32_t test::float_distance(float a, float b, float min_signal)
 
 uint64_t test::float_distance(double a, double b, double min_signal)
```

Comment:
@gcwenger Can you take a look at the changes to `close_f` and `float_distance`? (Rest of the PR looks fine to me.)